modules/developer_manual/pages/core/apis/ocs/user-sync-api.adoc

```diff
@@ -0,0 +1,95 @@
+= OCS User Sync API
+:toc: right
+:toclevels: 1
+:request-base-path: ocs/v2.php
+
+== Introduction
+
+This endpoint triggers user-sync for a specific user.
+
+* Path: `{request-base-path}/cloud/user-sync/{userid}`
+* Method: `POST`
+
+== Requirements
+
+This endpoint can only be executed by a user with admin privileges. 
+We suggest creating a technical user who is in the admin group to run this command with.
+
+== Request Parameters
+
+[cols="15%,15%,70%",options="header",width=100%]
+|===
+|Attribute 
+|Type 
+|Description
+
+|`userid` 
+|string 
+|The id of the user to trigger a sync for.
+|===
+
+== Returns
+
+The request returns an:
+
+* `HTTP 200 OK` - If the user sync was executed.
```

Comment:
Have not seen it earlier, but it woud be easier to read if the results are put into a table

Reply:
Sure. Fair enough.